eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,280 +1,314 @@
+                                                                                  record.trait_refs_list,
+                                                                                  report)
+                elif allele_origin == 'somatic':
+                    evidence_string = evidence_strings.CTTVSomaticEvidenceString(trait.efo_list,
+                                                                                 record.clin_sig,
+                                                                                 record.clinvarRecord,
+                                                                                 ensembl_gene_id,
+                                                                                 ensembl_gene_id_uri,
+                                                                                 record.measure_set_refs_list,
+                                                                                 record.observed_regs_list,
+                                                                                 trait.trait_counter,
+                                                                                 record.trait_refs_list,
+                                                                                 record.con_type,
+                                                                                 report)
+                add_evidence_string(record, evidence_string, report)
+                report.evidence_list.append(
+                    [record.clinvarRecord.accession, record.rs, ','.join(trait.clinvar_trait_list),
+                     ','.join(trait.efo_list)])
+                report.counters["n_valid_rs_and_nsv"] += (record.clinvarRecord.get_nsv(mappings.rcv_to_nsv) is not None)
+                report.counters["n_more_than_one_efo_term"] += (len(trait.efo_list) > 1)
+                report.traits.update(set(trait.efo_list))
+                report.ensembl_gene_id_uris.add(ensembl_gene_id_uri)
+
+        if record.n_ev_strings_per_record > 0:
+            report.counters["n_processed_clinvar_records"] += 1
+            if record.n_ev_strings_per_record > 1:
+                report.counters["n_multiple_evidence_strings"] += 1
+
+    report.write_output(dir_out)
+
+    print(report)
+
+
+def get_mappings(efo_mapping_file, ignore_terms_file, adapt_terms_file, snp_2_gene_file, variant_summary_file):
+    mappings = SimpleNamespace()
+    mappings.trait_2_efo, mappings.unavailable_efo_dict = load_efo_mapping(efo_mapping_file, ignore_terms_file,
+                                                                           adapt_terms_file)
+
+    mappings.consequence_type_dict = consequence_type.process_consequence_type_file(snp_2_gene_file)
+    mappings.rcv_to_rs, mappings.rcv_to_nsv = clinvar_record.get_rcv_to_rsnsv_mapping(variant_summary_file)
+
+    return mappings
+
+
+def get_record(cellbase_record, mappings, **kwargs):
```

Comment:
Having `kwargs` here doesn't seem very consistent with the way arguments are treated in the rest of the code. I think making them optional would be just enough.